### ISSUE TITLE
Parts for Sidewinder X1

### ISSUE THREAD

**Issue by acheljay** [MirageC79/Interfaces-for-WobbleX-integration#16]
Hi, could it be possible to have parts for the sidewinder X1, certainly one of the most wobbly printer I know ? Many thks in advance

**Comment by washo4evr** [comment]
would X1 parts fit the X2?

**Comment by acheljay** [comment]
I don't think cause the X2 have a new anti wobble system with spring tension for the wheels

**Comment by gt3rs40** [comment]
X1 parts would be awesome

**Comment by Caos-Maker89** [comment]
waiting for x1 parts :)

**Comment by fkahhaleh** [comment]
If anyone is still interested let me know. 
Close to trying it out and publishing on some sites.
[If you want to beta test let me know!]

![image](https://github.com/MirageC79/Interfaces-for-WobbleX-integration/assets/9301486/0068ca5e-d0ec-4302-853e-340b2a03249c)

**Comment by acheljay** [comment]
yup always interrested cause I did'nt find any good one for the X1

**Comment by gt3rs40** [comment]
Still interested :)

**Comment by washo4evr** [comment]
can this be ported to the X2? How different are the 2 models?
I just received my wobbleX yesterday but not sure how to install on my X2

**Comment by fkahhaleh** [comment]
Alright
So Amazon didn't deliver the parts today as planned. Instead of keeping you guys from potentially trying it out.
Here is a link to [Thingiverse](https://www.thingiverse.com/thing:6144103).

It is heavily a WIP/concept that I have not tried.  Keep that in mind :)

Feedback is welcome of course!

**Comment by fkahhaleh** [comment]
Perhaps a quick update on my end.
I was surprised that my Sidewinder X1 (v4) already had a similar mechanism to address Z-banding. (basically a blue cube with the brass nut inside with some degree of freedom for movement)
But I did install one side and I noticed that when there is z-axis movement, the rotation actually gets the Wabble-X moving too much that it actually almost jumps the metal ball bearings. Perhaps this is similar to the weight distribution issue mentioned in the wiki.

I ended up replacing the original parts for the X1 because the above design I provided did not work!

**Comment by ApolloF** [comment]
You can just use the mockup parts for the x1.
